packages/gatsby-theme-newrelic/src/components/DarkModeToggle.js

```diff
@@ -15,6 +15,12 @@ const DarkModeToggle = ({ className, size, onClick }) => {
       onClick={(e) => {
         darkMode.toggle();
 
+        const mode = darkMode.value ? 'dark' : 'light';
+
+        if (window.newrelic) {
+          window.newrelic.setCustomAttribute('mode', mode);
```

Comment:
```suggestion
        if (window.newrelic) {
          window.newrelic.setCustomAttribute('mode', darkMode.value ? 'dark' : 'light');
```

Very minor nit, we could save a couple lines be inlining the value here.